lib/elixir/lib/enum.ex

```diff
@@ -2129,17 +2132,22 @@ defmodule Enum do
   end
 
   @doc """
-  Returns a subset list of the given enumerable, from `range.first` to `range.last` positions.
+  Returns a subset list of the given enumerable, from `index_range.first` (zero-based) to
+  `index_range.last`.
 
-  Given `enumerable`, it drops elements until element position `range.first`,
-  then takes elements until element position `range.last` (inclusive).
+  `index_range` must be a `(range)[t:Range.t/0]`.
 
-  Positions are normalized, meaning that negative positions will be counted from the end
+  Given `enumerable`, it drops elements before `index_range.first`,
```

Comment:
What do you think about rewording the first part "Given `enumerable`", to "Given an `enumerable`" or:

> **On the** given `enumerable`, it drops elements before `index_range.first`, then takes elements until element `index_range.last` (inclusively).

Reply:
done